test/e2e-matched-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname "$0")/e2e-common.sh
+export BROKER_TEMPLATES=./templates/kafka-broker
```

Comment:
I wouldn't hardcode the template but default it

```suggestion
export BROKER_TEMPLATES="${BROKER_TEMPLATES:-./templates/kafka-broker}"
```

Reply:
In addition, the default path should take into account that the `pwd` is not necessarily the root directory, like if I run the `hack/run.sh` script when the `pwd` is the `hack` directory, this template path will be invalid